.env.example

```
@@ -11,3 +11,4 @@ VITE_SWAP_BUTTON_SWITCH="true"
 VITE_PAYMASTER_URL="http://localhost:5050"
 VITE_SENTRY_ENVIRONMENT="staging"
 VITE_PULSE_NODE_URL="https://pulse.etherspot.io"
+MOBULA_API_KEY="your_api_key_here"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)